bip-update-process.md

```diff
@@ -0,0 +1,646 @@
+* Motivation -- The motivation is critical for BIPs. It should clearly explain what issue the BIP addresses, and how the
+  existing situation is inadequate to address the problem that the BIP solves.
+* Specification -- The technical specification should describe the syntax and semantics of any new feature. The
+  specification should be detailed enough to enable any bitcoin project to create an interoperable implementation.
+* Rationale -- The rationale fleshes out the specification by describing what motivated the design and why particular
+  design decisions were made. It should describe alternate designs that were considered and related work. The rationale
+  should address important objections or concerns raised.
+* Backward compatibility -- BIPs must include a section describing any backwards incompatibilities and their severity.
+  The BIP must explain how implementers/users should deal with these incompatibilities.
+* Reference implementation -- Where applicable, a reference implementation, test vectors, and documentation must be
+  completed before the BIP can be given the status "Proposed". Any or all of these can be provided either in the BIP or as
+  auxiliary files.
+* Changelog -- A section to track modifications to a BIP after reaching Proposed status.
+* Copyright -- The BIP must be placed under an acceptable license ([see BIP Licensing below](#bip-licensing)).
+
+#### BIP header preamble
+
+Each BIP must begin with an RFC 822-style header preamble. The headers must appear in the following order. Headers
+marked with "\*" are optional. All other headers are required. The overview is followed by an explanation for each header.
+
+```
+  BIP: <BIP number, or "?" before being assigned>
+* Layer: <Consensus (soft fork) | Consensus (hard fork) | Peer Services | API/RPC | Applications>
+  Title: <BIP title, up to 50 characters>
+  Author: <list of authors’ names and email addresses>
+  Status: <Draft | Proposed | Active | Abandoned>
+  Type: <Specification | Informational | Process>
+  Created: <date the BIP was assigned a number, in ISO 8601 (yyyy-mm-dd) format>
+  License: <SPDX License Identifier of approved license(s)>
+* License-Code: <SPDX License Identifier for Code under different approved license(s)>
+* Discussion: <list of mailing list threads, or other noteworthy discussions in "date: link" format>
+* Revision: Version number of this document after a Changelog has been introduced
+* Requires: <BIP number(s)>
+* Replaces: <BIP number>
+* Superseded-By: <BIP number>
+```
+
+__BIP__: BIP number, or "?" before a number was assigned. The proposal must not self-assign a BIP number. Numbers are
+assigned by BIP Editors.
+
+__Layer__: The Layer header documents which layer of bitcoin the BIP applies to. See [BIP 123](bip-0123.mediawiki) for
+definitions of the various BIP layers.
+
+__Title__: A short descriptive title. Strongly preferred to be less than 50 characters long.
```

Comment:
nit: Is there a reason for not just saying "Must be" instead of "Strongly preferred to be"? This would also be more consistent with the "up to 50 characters" wording in the other parts of the BIP.

Reply:
Sure, taken.